src/components/product-grid/product-grid.js

```diff
@@ -10,7 +10,14 @@ function init() {
   componentEl = document.querySelector('[data-component="grid"]');
 }
 function update(products){
-  componentEl.innerHTML = renderTiles({products});
+  if(products){
+    if(componentEl.style.display = 'block')
+      componentEl.style.display = 'grid';
```

Comment:
minor style comment—not a blocker—I find it a little clearer and safer to only use 'bracketless' if statements when the code all fits on one line. If the code wraps like it does here, I would recommend using brackets.

Reply:
I have updated.